data/migrations/V0098__add_ofec_sched_a_agg_state_vw.sql

```diff
@@ -0,0 +1,32 @@
+DROP VIEW IF EXISTS public.ofec_sched_a_agg_state_vw;
+
+CREATE OR REPLACE VIEW public.ofec_sched_a_agg_state_vw AS
+select cmte_id,
```

Comment:
Can we make the layout easier to read and more consistent? I don't know if we ever completed the work @hcaofec was doing on database code conventions. Let's start with uppercase keywords, lowercase for all else, indented SELECT list, indented JOIN conditions, etc.